app/_hub/kong-inc/ai-proxy/how-to/_streaming.md

```diff
@@ -0,0 +1,64 @@
+---
+nav_title: Streaming
+title: Configure Streaming with AI Proxy
+---
+
+This guide walks you through setting up the AI Proxy plugin with streaming.
+
+{% include_cached /md/plugins-hub/ai-providers-prereqs.md snippet='intro' %}
+
+## How it works
+
+Streaming is a mode where a client can specify `"stream": true` in their request and the LLM server will stream each piece of the response text (usually token-by-token) as a server-sent event.
+
+Kong captures each batch of events and translates them back into the inference format, so that all providers are compatible with the same framework that you create on your side.
+
+When streaming is disabled, requests proxy directly to the LLM look like this:
+
+{% mermaid %}
+sequenceDiagram
+    actor Client
+    participant Kong
+    Note right of Kong: AI Proxy plugin
+    Client->>+Kong: 
+    Kong->>+Cloud LLM: Sets proxy request information
+    Cloud LLM->>+Client: Sends chunk to client
+{% endmermaid %}
```

Comment:
@lena-larionova Is there a better way to configure the mermaid diagram I made so it looks more like the original?

Original:
![317228104-9b9b41ff-4cbb-4512-bf04-06c2092b573b](https://github.com/Kong/docs.konghq.com/assets/75819066/f548fc3a-4f32-46da-b40d-0fbf869a1ff7)


Mine: https://deploy-preview-7293--kongdocs.netlify.app/hub/kong-inc/ai-proxy/how-to/streaming/

Reply:
If you use a flowchart, you can use something like this:

```
flowchart LR
  A(client)
  B(Kong Gateway with 
  AI Proxy plugin)
  C(Cloud LLM)

  A --> B
  B --sends request 
  information--> C
  C --> A
  ```